core/market/src/protocol/discovery.rs

```diff
@@ -112,7 +112,7 @@ impl Discovery {
         let bcast_msg = SendBroadcastMessage::new(OffersBcast { offer_ids });
 
         counter!("market.offers.broadcasts.net", 1);
-        gauge!("market.offers.broadcasts.size", size as i64);
+        value!("market.offers.broadcasts.len", size as u64);
```

Comment:
:+1: `.len` is more understandable for me than `.size`

Reply:
I had to change the name, because metrics server (Prometheus) has remembered `...broadcast.size` has `counter` type.